src/tcp-client-test/LineTcpSenderTests.cs

```diff
@@ -132,8 +132,10 @@ public async Task AuthFailBadKey()
         }
         catch (IOException ex)
         {
-            StringAssert.StartsWith("Unable to write data to the transport connection: ", ex.Message,
-                "Bad exception message");
+            // TODO: Verify that the exception message is correct
```

Comment:
Is it possible to address this TODO or it's left for future purposes?

Reply:
Sorry, I overlooked the PR description. @ideoma could you review the PR and take a look at this error message in particular?

Reply:
If you look at the original message, you can see that I changed it, to "Unable to ....". But I don't know if my change is correct, or if there actually is a bug here